Provide unique module names for the generated measurement clients during multiple client generation

Jotheeswaran-Nandagopal opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Provide unique module names for the generated measurement clients by appending a numerical suffix (_1, _2, etc.) to the name during multiple client generation and interactive mode.

### Why should this Pull Request be merged?

- When creating multiple clients, one Python measurement client for the service class `ni.examples.SampleMeasurement_Python` and one LabVIEW measurement client for `ni.examples.SampleMeasurement_LabVIEW` will both be framed with the module name `sample_measurement_client`, leading to file overwriting.
- In this implementation, we will be appending a numerical suffix to provide unique module name to mitigate the above case.

**Context** - [Bug 2894546](https://dev.azure.com/ni/DevCentral/_workitems/edit/2894546)

### What testing has been done?

- Existing test cases passed.
- Tested manually.

jayaseelan-james commented on packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py:
```diff
@@ -103,21 +103,22 @@ def _create_client(
 
     print(
-        f"The measurement plug-in client for the service class '{measurement_service_class}' is created successfully."
+        f"The measurement plug-in client for the service class '{measurement_service_class}' is created successfully as '{module_name}.py'."
```
nit.
```suggestion
        f"The measurement plug-in client for the service class '{measurement_service_class}' has been successfully created as '{module_name}.py'."
```

Jotheeswaran-Nandagopal commented:
@bkeryan, I have updated the logic to check for module name repetition during multiple client generating and added suffix only during multiple clients generation. Kindly, help us reviewing the changes.

bkeryan commented on packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py:
Ok, this works.
```
(.venv) PS D:\dev\measurement-plugin-python\examples\client> ni-measurement-plugin-client-generator --all
...
The measurement plug-in client for the service class 'ni.examples.SampleMeasurement_NET' has been successfully created as 'sample_measurement_client.py'.
The measurement plug-in client for the service class 'ni.examples.SampleMeasurement_Python' has been successfully created as 'sample_measurement_client2.py'.
...
```

```
(.venv) PS D:\dev\measurement-plugin-python\examples\client> python
Python 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import sample_measurement_client
>>> c = sample_measurement_client.SampleMeasurementClient()
>>> c.measure()
Outputs(bool_output=False, float_output=0.6000000238418579, string_output='sample string', enum_output=<EnumConfigurationEnum.Blue: 5>, double_array_output=[0.1, 0.2, 0.3], string_array_output=['String1', 'String2'])
>>> import sample_measurement_client2
>>> d = sample_measurement_client2.SampleMeasurementClient()
>>> d.measure()
Outputs(float_out=0.05999999865889549, double_array_out=[0.1, 0.2, 0.3], bool_out=False, string_out='sample string', enum_out=<EnumInEnum.BLUE: 3>, protobuf_enum_out=<ProtobufEnumInEnum.BLACK: 3>, string_array_out=['String1', 'String2'])
```
